src/docker.rs

```diff
@@ -170,8 +176,25 @@ pub fn run(
 
     docker.arg("--rm");
 
-    if target.needs_docker_privileged() {
-        docker.arg("--privileged");
+    // docker uses seccomp now on all installations
+    if target.needs_docker_seccomp() {
+        // write to our seccomp profile if it doesn't exist.
+        let mut path = env::current_dir().wrap_err("couldn't get current directory")?;
+        path.push("target");
+        path.push(target.triple());
+        fs::create_dir_all(&path).wrap_err("couldn't make target directory")?;
+        path.push("seccomp.json");
```

Comment:
this should be a function, maybe place it in `file`

Reply:
I fixed it :)

Reply:
Ok perfect, I'll modify this then.

Reply:
Is that better now?